packages/terser/test/test.js

```diff
@@ -356,3 +358,21 @@ test.serial('terser preserve vars in nameCache when provided', async (t) => {
     }
   });
 });
+
+test.serial('minify when os.cpus().length === 0', async (t) => {
+  const original = os.cpus;
```

Comment:
Maybe the test needs to also make `os.availableParallelism` unavailable, otherwise it won't use `os.cpus`.